documentation/data-importer.adoc

```diff
@@ -330,166 +270,86 @@ As mentioned before, if any element in your model is dashed, you will **not** be
 
 == Running an import
 
-You have drawn your data model, mapped every single element, and checked the preview to make sure things connect the way you intended.
-That means you are ready to run the actual import.
-
-Click highlight:import/import-run-import-button[Run import] to import the files specified in your model.
-The simple model you have created in this example maps to three files.
-Any uploaded files not specified in your model are **not** imported, but they remain in the highlight:import/import-file-panel[Files panel].
+Click highlight:import/import-run-import-button[Run import] to import the files specified in your model. 
+The simple model you have created in this example only maps to 3 of the 11 available files.
+The files not specified in your model are **not** imported, but they remain in the highlight:import/import-file-panel[Files panel].
 
-If any mapping is incomplete, Data Importer will tell you that your model has errors and add a red exclamations mark next to the incomplete element in the model.
+If any mapping is incomplete, we will tell you that your model has errors and add a red exclamations mark is shown next to the incomplete element in the model.
```

Comment:
Extraneous "is shown" in "...we will tell you that your model has errors and add a red exclamations mark is shown next to the incomplete...".